org_chart.js

```diff
@@ -4,70 +4,42 @@ let elon = {
   position: "Boss",
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
-  }
-]
 
-// assign each a manager
-let [alan, grace, donald] = new_hires
+function Employee(name, email, position, project, manager){
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.project = project;
+  this.manager = manager;
```

Comment:
According to the original example, setting the manager should be a separate operation/function